docs/diagrams/StorageClassDiagram.puml

```diff
@@ -15,11 +15,14 @@ Class "<<interface>>\nStorage" as Storage
 Class StorageManager
 
 package "AddressBook Storage" #F4F6F6{
-Class "<<interface>>\nAddressBookStorage" as AddressBookStorage
-Class JsonAddressBookStorage
-Class JsonSerializableAddressBook
+Class "<<interface>>\nEduContactsStorage" as AddressBookStorage
+Class JsonEduContactsStorage
+Class JsonSerializableEduContacts
 Class JsonAdaptedPerson
-Class JsonAdaptedTag
+Class JsonAdaptedModule
+Class JsonAdaptedGrade
+Class JsonAdaptedPerson
```

Comment:
```suggestion
```
I think JsonAdaptedGrade can be removed from the diagram since the class has no usages.
I also think JsonAdaptedPerson can be removed since there is already a JsonAdaptedPerson class in the package (see line 21).